Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM gradle:7.4-jdk8-focal AS build
 WORKDIR /notification-plugin
 COPY . ./
-RUN gradle clean classes
+RUN gradle clean jar
+
+FROM eclipse-temurin:8-jre
```

Comment:
Just a question out of curiosity, fine as-is: Is there are reason why JDK 8 is used instead of, e.g. LTS17? As the JAR is only running inside a Docker container there should be no environment constraints.

Reply:
For parsing the XML files we need some dependencies (JAXB). I remember that when I initialized this plugin it was complicated to find the right dependencies as they were handled by a corresponding Jenkins dependency.
Therefore, I just used this version to make it working, but it is planned to update the Java version.